examples/jsm/misc/ProgressiveLightMap.js

```diff
@@ -0,0 +1,315 @@
+import * as THREE from '../../../build/three.module.js';
+import { potpack } from '../libs/potpack.module.js';
+
+/**
+ * Progressive Light Map Accumulator, by [zalo](https://github.com/zalo/)
+ *
+ * To use, simply construct a `ProgressiveLightMap` object,
+ * `plmap.addObjectsToLightMap(object)` an array of semi-static
+ * objects and lights to the class once, and then call
+ * `plmap.update(camera)` every frame to begin accumulating
+ * lighting samples.
+ *
+ * This should begin accumulating lightmaps which apply to
+ * your objects, so you can start jittering lighting to achieve
+ * the texture-space effect you're looking for.
+ *
+ * @param {WebGLRenderer} renderer A WebGL Rendering Context
+ * @param {number} res The side-long dimension of you total lightmap
+ */
+var ProgressiveLightMap = function ( renderer, res = 1024 ) {
```

Comment:
Do you mind writing this code as a ES6 class?

Reply:
Interesting; I'm still learning when it comes to Modern Javascript Syntax, so hopefully I did that right...

Reply:
It looks good! 👍